docs/Incident.md

```diff
@@ -0,0 +1,30 @@
+# Incident
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Threshold** | **float64** |  | [optional] 
+**Status** | **string** |  | [optional] 
+**StartedAt** | **string** |  | [optional] 
+**Severity** | **string** |  | [optional] 
+**SampleSizeUnit** | **string** |  | [optional] 
+**SampleSize** | **float64** |  | [optional] 
+**ResolvedAt** | **string** |  | [optional] 
+**Notifications** | [**[]IncidentNotification**](IncidentNotification.md) |  | [optional] 
+**NotificationRules** | [**[]IncidentNotificationRule**](IncidentNotificationRule.md) |  | [optional] 
+**Measurement** | **string** |  | [optional] 
+**MeasuredValueOnClose** | **float64** |  | [optional] 
+**MeasuredValue** | **float64** |  | [optional] 
+**IncidentKey** | **string** |  | [optional] 
+**Impact** | **string** |  | [optional] 
+**Id** | **string** |  | [optional] 
+**ErrorDescription** | **string** |  | [optional] 
+**Description** | **string** |  | [optional] 
+**Breakdowns** | [**[]IncidentBreakdown**](IncidentBreakdown.md) |  | [optional] 
+**AffectedViewsPerHourOnOpen** | **float64** |  | [optional] 
+**AffectedViewsPerHour** | **float64** |  | [optional] 
+**AffectedViews** | **float64** |  | [optional] 
```

Comment:
Same as PHP, so I'm thinking this may just be openAPI, but this should actually be an integer.